### PR TITLE
734 help modal

### PR DESCRIPTION
**Pull request by bcolloran**
This is the MVP of a help panel. We can (and will) add content (links to api docs etc) as ewe begin to update our docs and migrate our notebooks from the iodide-examples repo to our server, and as we add content we'll clean up the presentation. This is basically just a skeleton for us to build off of at this point.

### CODE REVIEW

**Comment by bcolloran** [comment]
@wlach oh oops. you can use "h" in command mode, but i should also add a "help" item to the menu.

File: src/actions/actions.js [hunk 1]
```diff
@@ -403,17 +403,11 @@ export function deleteCell() {
...
-//   return {
-//     type: 'TOGGLE_EVAL_FRAME_VISIBILITY',
-//   }
-// }
-
-// export function toggleEditorVisibility() {
-//   return {
-//     type: 'TOGGLE_EDITOR_VISIBILITY',
-//   }
-// }
```
[the remark this answers is not in the document]
Comment (bcolloran) [reply src/actions/actions.js]: Gotcha. Do you mind if I leave this in in this case?

**Comment by bcolloran** [comment]
@wlach added a menu item for the help menu.

**Comment by bcolloran** [comment]
@teonbrooks how dos this look to you?

File: src/actions/actions.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (wlach) [reply src/actions/actions.js]: Oh yeah, no problem in this case.

**Comment by teonbrooks** [comment]
this is just what I wanted :) lgtm

thanks!

**Pull request merged**